### DISCUSSION TITLE
Support for Balochi

### DISCUSSION THREAD

**Discussion by strickvl** [explosion/spaCy#12706] [Language Support]
I've [created a branch](https://github.com/strickvl/spaCy/tree/balochi) to start adding in support for the Balochi language. Is it preferred that this comes in 'finished and ready to go', or is it worth making a draft PR to get the ball rolling on feedback? My initial contribution would be a minimal one with language data and then I'd hope to build on that.

**Comment by rmitsch** [comment]
Hi @strickvl, a draft PR is very much welcome!
